### DISCUSSION TITLE
SQL Message Response

### DISCUSSION THREAD

**Discussion by ghost** [oracle/python-oracledb#63] [Q&A] [answered]
Hi Team,

I was willing to get the name of table/procedure/package when we create/drop for the executed query.
Eg: res = cursoue.execute("CREATE TABLE <NAME>(COL DTYPE)")
      Response as (Table <Name> created)

Is this possible to get the message response? If yes, please help me.

Thanks in advance.

**Comment by anthony-tuininga** [comment, marked as answer]
This information is not provided by the database. You will have to parse the SQL that is being sent and use that information to generate the desired message. I did this in the [cx_OracleTools](https://github.com/anthony-tuininga/cx_OracleTools). That project is in the process of being converted to use python-oracledb -- but only in my spare time, which isn't significant at this point in my life!
